app/components/types_de_champ_editor/champ_component.rb

```diff
@@ -0,0 +1,115 @@
+class TypesDeChampEditor::ChampComponent < ApplicationComponent
+  def initialize(coordinate:, focus: nil)
+    @coordinate = coordinate
+    @focus = focus
```

Comment:
Je crois que tu peux directement passer un boolean, en mettant `true` ici `app/views/administrateurs/types_de_champ/_insert.turbo_stream.haml`

```suggestion
    @focused = focused
```